internal/store/pod.go

```diff
@@ -1362,6 +1364,32 @@ func createPodStartTimeFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodStatusContainersReadyTimeFamilyGenerator() generator.FamilyGenerator {
+    return *generator.NewFamilyGenerator(
+        "kube_pod_status_containers_ready_time",
+        "Readiness achieved time in unix timestamp for a pod containers.",
+        metric.Gauge,
+        "",
+        wrapPodFunc(func(p *v1.Pod) *metric.Family {
+            ms := []*metric.Metric{}
+
+			for _, c := range p.Status.Conditions {
+				if c.Type == v1.ContainersReady {
+                    ms = append(ms, &metric.Metric{
```

Comment:
This seems a bit off formating, did you run gofmt here?

Reply:
Forgot it! Mea maxima culpa. Fixed my mistake.